source/installation-guide/wazuh-dashboard/index.rst

```diff
@@ -9,8 +9,11 @@
 Wazuh dashboard
 ===============
 
+This central component is a flexible and intuitive web interface for mining, analyzing, and visualizing data. It provides out-of-the-box dashboards, allowing you to seamlessly navigate through the user interface. 
 
-This central component is a flexible and intuitive web interface for mining, analyzing, and visualizing data. It provides out-of-the-box dashboards, allowing you to seamlessly navigate through the user interface. Users can quickly visualize security events, detected vulnerable applications, file integrity monitoring data, configuration assessment results, cloud infrastructure monitoring events, and regulatory compliance, such as PCI DSS, GDPR, CIS, HIPAA, and NIST 800-53 standards. If you want to learn more about Wazuh components, check the :doc:`Getting started section <../../getting-started/components/index>`.
+Users can quickly visualize security events, detect vulnerable applications, file integrity monitoring data, configuration assessment results, cloud infrastructure monitoring events, and regulatory compliance, such as PCI DSS, GDPR, CIS, HIPAA, and NIST 800-53 standards. 
+
+If you want to learn more about Wazuh components, check the :doc:`Getting started <../../getting-started/components/index>` section.
```

Comment:
Changing _check_ to _see_ to avoid repetition with the following sentence. 

```suggestion
If you want to learn more about Wazuh components, see the :doc:`Getting started <../../getting-started/components/index>` section.
```